capplets/common/mate-theme-apply.c

```diff
@@ -48,13 +50,26 @@ void
 mate_meta_theme_set (MateThemeMetaInfo *meta_theme_info)
 {
   GSettings *interface_settings;
+  GSettings *interface_gnome_settings;
   GSettings *marco_settings;
   GSettings *mouse_settings;
   GSettings *notification_settings = NULL;
   gchar *old_key;
   gint old_key_int;
+  gboolean wayland;
+
+  /*Find out if we are running under wayland and have xwayland
+   *Sending signal 0 with killall doesn't kill the process but
+   *the return value tells us whether the process exists
+   */
+  if (GDK_IS_X11_DISPLAY ((gdk_display_get_default())))
+    wayland = FALSE;
```

Comment:
The comment seems out of sync.
But I like the check better than the system call :)

Reply:
Thanks for catching this, looks like two code blocks I wrote at two different times used two different methods

Reply:
That is fixed, I missed it in the previous commit that removed the same system call elsewhere